### PR TITLE
Update language_de.h

### PR DESCRIPTION
**Pull request by blubbfish**
Add the latest language strings

### CODE REVIEW

File: Marlin/language_de.h [hunk 1]
```diff
@@ -146,6 +146,8 @@
 #define MSG_INIT_SDCARD                     "SD-Karte erkennen"  // Manually initialize the SD-card via user interface
 #define MSG_CNG_SDCARD                      "SD-Karte getauscht" // SD-card changed by user. For machines with no autocarddetect. Both send "M21"
 #define MSG_ZPROBE_OUT                      "Sensor ausserhalb"
+#define MSG_BLTOUCH_SELFTEST                "Bluetooth Test"
```
Comment (Kaibob2) [inline Marlin/language_de.h]: BLTouch is a probe for bed leveling. The correct string must be "BLTouch Selbsttest"

File: Marlin/language_de.h [hunk 2]
```diff
@@ -146,6 +146,8 @@
...
+#define MSG_BLTOUCH_RESET                   "Reset BLTouch"
```
Comment (Kaibob2) [inline Marlin/language_de.h]: For consistency i woudl prefer "BLTouch Reset"

File: Marlin/language_de.h [hunk 3]
```diff
@@ -203,6 +205,9 @@
 #define MSG_INFO_MIN_TEMP                   "Min Temp"
 #define MSG_INFO_MAX_TEMP                   "Max Temp"
 #define MSG_INFO_PSU                        "Stromversorgung"
+#define MSG_DRIVE_STRENGTH                  "Motorkraft"
```
Comment (Kaibob2) [inline Marlin/language_de.h]: "Antriebskraft" would be more precise

File: Marlin/language_de.h [hunk 4]
```diff
@@ -203,6 +205,9 @@
...
+#define MSG_DAC_PERCENT                     "Kraft %"
```
Comment (Kaibob2) [inline Marlin/language_de.h]: "Antriebskraft in %" would be more precise

**Review by Kaibob2 — CHANGES_REQUESTED** [review]

File: Marlin/language_de.h (hunk 1 above)
Comment (blubbfish) [reply Marlin/language_de.h]: BLTouch Selbsttest
123456789123456789

Ok thats not to long as i thought. So thats fine :)

File: Marlin/language_de.h (hunk 2 above)
Comment (blubbfish) [reply Marlin/language_de.h]: I think there is no difference, but your version sounds nicer, so I will take that

File: Marlin/language_de.h (hunk 3 above)
Comment (blubbfish) [reply Marlin/language_de.h]: Your right, but i dont see another method to drive the printer without using a motor

File: Marlin/language_de.h (hunk 2 above)
Comment (Kaibob2) [reply Marlin/language_de.h]: There IS no difference, it's only for looking nicer:
**BLTouch Selbsttest
BLTouch Reset**
looks better than
**BLTouch Selbsttest
Reset BLTouch**

File: Marlin/language_de.h (hunk 3 above)
Comment (Kaibob2) [reply Marlin/language_de.h]: Maybe with a crank? ;)

File: Marlin/language_de.h [hunk 5]
```diff
@@ -203,6 +205,9 @@
 #define MSG_INFO_MIN_TEMP                   "Min Temp"
 #define MSG_INFO_MAX_TEMP                   "Max Temp"
 #define MSG_INFO_PSU                        "Stromversorgung"
+#define MSG_DRIVE_STRENGTH                  "Antriebskraft"
+#define MSG_DAC_PERCENT                     "Antriebskraft in %"
```
Comment (AnHardt) [inline Marlin/language_de.h:209]: Zu lang!
Da muß noch der Motorname davor und der Wert dahinter.
```
      MENU_ITEM_EDIT_CALLBACK(int3, MSG_X " " MSG_DAC_PERCENT, &driverPercent[X_AXIS], 0, 100, dac_driver_commit);
      MENU_ITEM_EDIT_CALLBACK(int3, MSG_Y " " MSG_DAC_PERCENT, &driverPercent[Y_AXIS], 0, 100, dac_driver_commit);
      MENU_ITEM_EDIT_CALLBACK(int3, MSG_Z " " MSG_DAC_PERCENT, &driverPercent[Z_AXIS], 0, 100, dac_driver_commit);
      MENU_ITEM_EDIT_CALLBACK(int3, MSG_E " " MSG_DAC_PERCENT, &driverPercent[E_AXIS], 0, 100, dac_driver_commit);
``` 

"Treiber %" oder "Motor %".

File: Marlin/language_de.h [hunk 6]
```diff
@@ -203,6 +205,9 @@
 #define MSG_INFO_MIN_TEMP                   "Min Temp"
 #define MSG_INFO_MAX_TEMP                   "Max Temp"
 #define MSG_INFO_PSU                        "Stromversorgung"
+#define MSG_DRIVE_STRENGTH                  "Antriebskraft"
```
Comment (AnHardt) [inline Marlin/language_de.h:208]: Es geht um "Motorströme".

**Comment by thinkyhead** [comment]
This, or #5359?

File: Marlin/language_de.h [hunk 7]
```diff
@@ -203,6 +205,9 @@
...
+#define MSG_DAC_PERCENT                     "Antriebskraft in %"
+#define MSG_DAC_EEPROM_WRITE                "DAC EEPROM Schreiben"
```
Comment (AnHardt) [inline Marlin/language_de.h:210]: Ideally <= 16.

**Pull request closed without merging**